packages/container/lib/registry.js

```diff
@@ -732,31 +732,31 @@ Registry.prototype = {
   getInjections(fullName) {
     let injections = this._injections[fullName] || [];
     if (this.fallback) {
-      injections = injections.concat(this.fallback.getInjections(fullName));
+      injections.push(...this.fallback.getInjections(fullName));
```

Comment:
before/after is not same here, so each time you call `getInjections` it it modifies `this._injections[fullName]` if there is fallback.  am I wrong ? @stefanpenner @artemgurzhii